### PR TITLE
Issue 11 setup usernames

### PR DESCRIPTION
**Pull request by NathanSegerlind**
This sets up 'oni' as the default user for setup, per issue #11.

This change must be taken along with the analogous change to ml-ops (because of dependencies on oni.conf in the install_ml.sh and ml_ops.sh)

### CODE REVIEW

**Comment by NathanSegerlind** [comment]
These have been tested on FM16; ml_ops.sh runs to completion.

File: hdfs_setup.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 DSOURCES=('flow' 'dns')
 DFOLDERS=('binary' 'csv' 'hive' 'stage')
-DBNAME='duxbury'
-source /etc/duxbay.conf
+DBNAME='oni'
+source /etc/oni.conf
```
Comment (natedogs911) [inline hdfs_setup.sh:7]: I think we should add "export DBNAME" here and use the oni.conf file to set this as it's taking the variable from line 5 instead.

the same could be said for DSOURCES and DFOLDERS

**Comment by natedogs911** [comment]
LGTM

**Comment by natedogs911** [comment]
Can you re-open your PR against 1.0.1 @NathanSegerlind

**Pull request closed without merging**